src/ucp/wireup/wireup_cm.c

```diff
@@ -210,6 +215,7 @@ static void uct_wireup_cm_tmp_ep_cleanup(ucp_ep_h tmp_ep, ucs_queue_head_t *queu
 
     ucs_trace("deleting tmp_ep %p", tmp_ep);
     ucp_ep_delete(tmp_ep);
+    tmp_ep = NULL;
```

Comment:
has no effect - it only changes local var
need to set cm_wireup_ep->tmp_ep to NULL